minio.json

```
@@ -29,23 +34,20 @@
                     }
                 },
                 "cmd_arguments": [
-                    [
-                        "server",
-                        "/data"
-                    ]
-                ],
+                    [ "server" , "/data" ]
+                 ],
                 "environment": {
-                    "MINIO_ACCESS_KEY": {
-                        "description": "MinIO Access Key (admin username), 1 to 128 alphanumeric characters",
-                        "label": "MinIO Access Key"
+                    "MINIO_ROOT_USER": {
+                        "description": "MinIO Administrator Username, 1 to 128 alphanumeric characters",
```

```suggestion
                        "description": "MinIO Administrator Username, 3 to 128 alphanumeric characters",
```

See also my comment regarding a reported minimum on the password also:
'''
Apr 06 14:26:36 rleap15-3 minio/minio:latest/MinIO[13506]:         Access key length should be at least 3, ...
'''